propagator/google_cloud_platform/lib/opentelemetry/propagator/google_cloud_platform/version.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# OpenTelemetry is an open source observability framework, providing a
+# general-purpose API, SDK, and related tools required for the instrumentation
+# of cloud-native software, frameworks, and libraries.
+#
+# The OpenTelemetry module provides global accessors for telemetry objects.
+# See the documentation for the `opentelemetry-api` gem for details.
+module OpenTelemetry
+  # Namespace for OpenTelemetry propagator extension libraries
+  module Propagator
+    # Namespace for OpenTelemetry GoogleCloudPlatform propagation
+    module GoogleCloudPlatform
+      VERSION = '0.0.1'
```

Comment:
Our release tooling will automatically set the version. no need to set it here.
```suggestion
      VERSION = '0.0.0'
```